### DISCUSSION TITLE
Cloudwatch: Alert List - Display cloudwatch alarms (NOT statistics)

### DISCUSSION THREAD

**Discussion by jjshoe** [grafana/grafana#41815] [Data Sources]
In cloudwatch, you can have panels which display various drawings of data, a table of logs, and you can include relevant alarms.

Stated another way, given web app hello world, I can have three panels:
1) Requests per second as a line graph
2) Last 100 log messages
3) Alarms for cpu, disk space, and requests per second

In Grafana, I would love support for #3. The visualization for "Alert List" does not seem to support querying AWS cloudwatch in any way.

Supporting requests:
https://github.com/grafana/grafana/issues/19013
https://community.grafana.com/t/show-aws-cloudwatch-alarms-in-table/14879

**Comment by sunker** [comment]
The annotation editor for CloudWatch returns alarms for a given metric and plots them as annotations in the panel. Not exactly what you're looking for but might be useful? 

Read more about annotations in grafana [here](https://grafana.com/docs/grafana/latest/dashboards/annotations/)

**Comment by jjshoe** [comment]
@sunker a list of alarms, and their health, "Good" or "Bad". I imagine this is exactly what alert list, though I don't know, as I couldn't configure it to work with aws.

AWS cloudwatch dashboards let you add a panel listing alarms, and their status. 

Annotations are not what I'm looking for. I'm not looking to graph or plot anything.

**Comment by sunker** [comment]
I see what you mean. Let's keep this feature request open for a while and see if there's more users that shows interest in this.

**Comment by peppermor** [comment]
I would love to see this possible with Cloudwatch. I have dashboard showing state off all monitoring components: Alerts in Icinga, Alertmanager, Grafana what I am missing is alarm list from Cloudwatch

**Comment by sarahzinger** [comment]
Hello,

As part of our work to move from [discussions to issues](https://github.com/grafana/grafana/discussions/73424), we're going through old discussions and seeing which we can close and which we can turn into issues. I've created this [github issue](https://github.com/grafana/grafana/issues/77962) from this discussion along with my understanding of the problem, if I've misrepresented anything or you have other thoughts to add feel free to comment there!

Thanks so much for your suggestion!
